ReactCommon/cxxreact/SampleCxxModule.cpp

```diff
@@ -114,6 +114,25 @@ auto SampleCxxModule::getMethods() -> std::vector<Method> {
         sample_->hello();
         return nullptr;
       }, SyncTag),
+    Method("addIfPositiveAsPromise", [this](dynamic args, Callback cb, Callback cbError) {
```

Comment:
Looks like the capture of `this` is actually not used, failing the build internally:

```
react-native-github/ReactCommon/cxxreact/SampleCxxModule.cpp:117:39: error: lambda capture 'this' is not used [-Werror,-Wunused-lambda-capture]
    Method("addIfPositiveAsPromise", [this](dynamic args, Callback cb, Callback cbError) {
                                      ^
react-native-github/ReactCommon/cxxreact/SampleCxxModule.cpp:126:37: error: lambda capture 'this' is not used [-Werror,-Wunused-lambda-capture]
    Method("addIfPositiveAsAsync", [this](dynamic args, Callback cb, Callback cbError) {
                                    ^
```

Reply:
Sure I can remove that.